docker-contributor/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG ARCH=
-FROM ${ARCH}ubuntu:focal
+FROM "${ARCH}ubuntu:focal"
```

Comment:
Is this tested and works? Because this is not a shell script, so I don't know what the allowed syntax is.

Reply:
No, the pipeline does not pick this up yet, so this was 1 part where I want the pipeline to test it.

Reply:
I don't mean the shellcheck pipeline, but the actual docker image creation with this file.

Reply:
Yes I was aware,

What I meant was: https://github.com/DOMjudge/domjudge-packaging/blob/main/.gitlab-ci.yml#L88 to test the building of containers for the {gitlabci,contributor,domserver,judge}image, but those only trigger if the HEAD commit has the change (instead of any commit in the PR)

Reply:
@eldering yes it seems to work:
https://gitlab.com/DOMjudge/domjudge-packaging/-/jobs/1904652587#L197

We correctly grab the correct architecture.

Reply:
If I interpret the manual correctly its also promised behavior:
https://docs.docker.com/engine/reference/builder/#environment-replacement